README.md

```diff
@@ -47,6 +47,24 @@ by executing `./enable-debug.sh https://svgomg.firebaseapp.com`.
 To debug a different PWA, execute the script with a different host:
 `./enable-debug.sh https://example.com`
 
+### Wrapping Up
```

Comment:
Maybe we could rename this section to "Debugging Digital Asset Links", and tweak the text below to reflect this change. How does that sound?

Reply:
Yes, sound's good.. what about  now?